### PR TITLE
Prepare for episode 36

### PR DESCRIPTION
**Pull request by HangingClowns**
@jonhoo for the coming Friday: Sept 10th

### CODE REVIEW

File: _episodes/036-luca-palmieri.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+title: "From Zero to Production"
+date: 2021-09-10T15:30:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e036-luca-palmieri.mp3
+duration: "1:07:42"
+length: "65103280"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+#  From Zero to Production  
+
+Allen Wyma talks with Luke Palmieri, a principal engineer, also the book author of ["Zero To Production in Rust"](https://www.zero2prod.com/index.html)
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:11]: Would be good to say _where_ they're a principal engineer. "Principal engineer" on its own doesn't say much.

Comment (jonhoo) [reply _episodes/036-luca-palmieri.md:11]: Also, this sentence is missing a full stop at the end.

File: _episodes/036-luca-palmieri.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Highlights
+
+- "Zero To Production In Rust" is an introduction of backend development in Rust.
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:24]: This doesn't seem important. I would delete it.

File: _episodes/036-luca-palmieri.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Highlights
+
+- "Zero To Production In Rust" is an introduction of backend development in Rust. 
+
+## Resources 
+- [Hexagonal Architecture](https://en.wikipedia.org/wiki/Hexagonal_architecture_(software))
+- [Asynchronous Programming in Rust](https://rust-lang.github.io/async-book/08_ecosystem/00_chapter.html)
+- [Rust for Rustaceans](https://nostarch.com/rust-rustaceans)
+- [Rust in Action](https://www.manning.com/books/rust-in-action)
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:30]: I'd prefer for these resources to be linked under the appropriate timed section below where possible. If a resource is referenced many times, then I'd be okay with listing it separately, though I'd prefer for the resources link list to be placed below the timestamps.

File: _episodes/036-luca-palmieri.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Timestamps 
+- [@02:30] - Inspiration
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:33]: Inspiration for what?

File: _episodes/036-luca-palmieri.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+- [@13:20] - Reason of using Rust
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:34]:
```suggestion
- [@13:20] - Reasons for using Rust [in what?]
```

File: _episodes/036-luca-palmieri.md (hunk 1 above)
Comment (jonhoo) [reply _episodes/036-luca-palmieri.md:11]: Looking at the timestamps, it seems like this is really an interview _about_ the book. That should be clear in the description here.

File: _episodes/036-luca-palmieri.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+- [Rust in Action](https://www.manning.com/books/rust-in-action)
+
+## Timestamps 
+- [@02:30] - Inspiration
+- [@13:20] - Reason of using Rust 
+- [@16:50] - [Actix Web](https://github.com/actix/actix-web)
+- [@32:21] - Challenges in using Rust 
+- [@36:30] - [krustlet](https://krustlet.dev/)
+- [@46:35] - Process of writing the book 
+- [@54:50] - Rewriting in Rust edition 2021?
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:39]: I'm not sure what this is trying to say?

File: _episodes/036-luca-palmieri.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+- [@57:40] - Rust's Community
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:40]: Any particular reason why the C is capitalized here?

File: _episodes/036-luca-palmieri.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+## Timestamps 
+- [@02:30] - Inspiration
+- [@13:20] - Reason of using Rust 
+- [@16:50] - [Actix Web](https://github.com/actix/actix-web)
+- [@32:21] - Challenges in using Rust 
+- [@36:30] - [krustlet](https://krustlet.dev/)
+- [@46:35] - Process of writing the book 
+- [@54:50] - Rewriting in Rust edition 2021?
+- [@57:40] - Rust's Community 
+- [@1:01:34] - Tips for Beginner
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:41]:
```suggestion
- [@1:01:34] - Tips for beginners
```

File: _episodes/036-luca-palmieri.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
+---
+title: "From Zero to Production"
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:2]: Let's mention Luke's name in the title too.

**Review by jonhoo — CHANGES_REQUESTED** [review]

File: _episodes/036-luca-palmieri.md (hunk 9 above)
Comment (jonhoo) [reply _episodes/036-luca-palmieri.md:2]: I'd still like this to be update.

Comment (HangingClowns) [reply _episodes/036-luca-palmieri.md:2]: Any suggestions?

Comment (jonhoo) [reply _episodes/036-luca-palmieri.md:2]: > From Zero to Production with Luca Palmieri

File: _episodes/036-luca-palmieri.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
+---
+title: "From Zero to Production with Luca Palmieri"
+date: 2021-09-10T15:30:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e036-luca-palmieri.mp3
+duration: "1:07:42"
+length: "65103280"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+#  From Zero to Production with Luca Palmieri  
+
```
Comment (jonhoo) [inline _episodes/036-luca-palmieri.md:10]:
```suggestion
```
We don't generally repeat the episode title in the description.

**Review by jonhoo — APPROVED** [review]

**Pull request merged**